site/app/models/gradeable/TaGradedGradeable.php

```diff
@@ -48,9 +48,14 @@ public function __construct(Core $core, GradedGradeable $graded_gradeable, array
         $this->graded_gradeable = $graded_gradeable;
 
         $this->setIdFromDatabase($details['id'] ?? 0);
-        $this->setOverallComment($details['overall_comment'] ?? '');
         $this->setUserViewedDate($details['user_viewed_date'] ?? null);
 
+        if (array_key_exists("overall_comments", $details)) {
+            foreach ($details['overall_comments'] as $commenter => $comment) {
+                $this->overall_comment[$commenter] = $comment;
+            }
```

Comment:
Can't this be simplified to just `$this->overall_coment = $details['overall_comments'];`

Reply:
Good catch. Simplified.